src/snyk/advisor/services/advisorApiClient.ts

```diff
@@ -47,11 +47,7 @@ export class AdvisorApiClient implements IAdvisorApiClient {
     const token = await this.configuration.getToken();
     this.http.interceptors.request.use(req => {
       req.baseURL = this.configuration.baseApiUrl;
-      req.headers = {
-        ...req.headers,
-        Authorization: `token ${token}`,
-      } as { [header: string]: string };
-
+      req.headers['Authorization'] = `token ${token}`;
```

Comment:
Change needed doe to Axios upgrade. This change sets the `Authorization` header individually instead of overwriting the entire headers object.

Reply:
suggestion: if this is the only place where axios is used, we may be able to get rid of the functionality entirely. I think advisor is a remnant of an incomplete contribution, which we need to remove anyway. Have you found any location where it is used in the extension right now?